src/sentry/integrations/slack/message_builder/help.py

```diff
@@ -8,7 +8,7 @@
 DM_COMMAND_HEADER = "*Direct Message Commands:*"
 CHANNEL_COMMANDS_HEADER = "*Channel Commands:*"
 CONTACT_HEADER = "*Contact:*"
-GENERAL_MESSAGE = "Just want to learn more about Sentry? Check out our <https://docs.sentry.io/product/alerts-notifications/alerts/|documentation>."
+GENERAL_MESSAGE = "Just want to learn more about Sentry? Check out our <https://docs.sentry.io/product/integrations/notification-incidents/slack/>."
```

Comment:
Looks like it's missing the "documentation" text.

Reply:
lol oops